classes/class-wpml-mu-domain-mapping-requirements.php

```diff
@@ -33,6 +33,11 @@ private function check_required_plugins() {
 			);
 		}
 
+		// This works too for 'Domain Mapping' from WPMUDEV.
+		if ( $this->is_plugin_active_for_network( 'domain-mapping/domain-mapping.php' ) ) {
```

Comment:
@dgwatkins as usual, the plugin folder might not be what we expect, even for must-use plugins.

What about checking if the `Domainmap_Plugin` class exists instead?